internal/impl/devbox.go

```diff
@@ -682,6 +681,47 @@ func (d *Devbox) printPackageUpdateMessage(
 	return nil
 }
 
+func (d *Devbox) computeNixEnv() ([]string, error) {
```

Comment:
Should this be called in other places we need env? (`Shell` and `RunScriptInNewNixShell`)

Reply:
I believe that is the intention (in a future PR)

Reply:
Correct!